pkgs/top-level/all-packages.nix

```diff
@@ -6708,18 +6708,18 @@ with pkgs;
 
   jython = callPackage ../development/interpreters/jython {};
 
```

Comment:
Can you also add aliases to `pkgs/top-level/aliases.nix`?

Reply:
Nice, didn't know about that. Done.